data/wild/unlocked_unowns.asm

```diff
@@ -7,11 +7,13 @@ endr
 ENDM
 
 UnlockedUnownLetterSets:
-; entries correspond to wUnlockedUnowns bits
-	dw .Set_A_K ; ENGINE_UNLOCKED_UNOWNS_A_TO_K
-	dw .Set_L_R ; ENGINE_UNLOCKED_UNOWNS_L_TO_R
-	dw .Set_S_W ; ENGINE_UNLOCKED_UNOWNS_S_TO_W
-	dw .Set_X_Z ; ENGINE_UNLOCKED_UNOWNS_X_TO_Z
+; entries correspond to UNLOCKED_UNOWNS_* constants
+	table_width 2, UnlockedUnownLetterSets
+	dw .Set_A_K ; UNLOCKED_UNOWNS_A_TO_K_F
+	dw .Set_L_R ; UNLOCKED_UNOWNS_L_TO_R_F
+	dw .Set_S_W ; UNLOCKED_UNOWNS_S_TO_W_F
+	dw .Set_X_Z ; UNLOCKED_UNOWNS_X_TO_Z_F
+	assert_table_length NUM_UNOWN_SETS
 .End
```

Comment:
Hm, now that `NUM_UNOWN_SETS` exists, we don't need this; `cp UnlockedUnownLetterSets.End - UnlockedUnownLetterSets` can be `cp NUM_UNOWN_SETS` in engine/battle/core.asm. (Also on second thought "`NUM_UNLOCKED_UNOWN_SETS`" might be more immediately clear.)